src/index.ts

```diff
@@ -13,27 +20,46 @@ export interface SafeInfo {
 }
 
 export interface SafeListeners {
-  onSafeInfo: (info: SafeInfo) => any;
+  onSafeInfo: (info: SafeInfo) => void;
 }
 
-export enum FromSafeMessages {
-  ON_SAFE_INFO = 'ON_SAFE_INFO',
+interface CustomMessageEvent extends MessageEvent {
+  data: {
+    messageId: keyof FromSafeMessages;
+    data: FromMessageToPayload[keyof FromSafeMessages];
+  };
+}
+
+export const TO_SAFE_MESSAGES = {
+  SEND_TRANSACTIONS: 'SEND_TRANSACTIONS' as const,
```

Comment:
same here

Reply:
I couldn't make this interface work with an enum, this is why I changed it:
https://github.com/gnosis/safe-apps-sdk/pull/25/files#diff-f41e9d04a45c83f3b6f6e630f10117feR37

Reply:
If you can make it work I have nothing against it, but I also don't see any reasons to switch to enum

Reply:
https://stackoverflow.com/questions/39701524/using-enum-as-interface-key-in-typescript